tests/dummy/app/controllers/horizontal-bar.js

```diff
@@ -35,11 +35,16 @@ export default SlideController.extend({
 
   configuredMaxLabelWidth: 100,
 
-  maxLabelWidth: Ember.computed('hasMaxLabelWidth', 'configuredMaxLabelWidth', function() {
-    if (this.get('hasMaxLabelWidth')) {
-      return this.get('configuredMaxLabelWidth');
-    } else {
-      return null;
+  maxLabelWidth: Ember.computed('hasMaxLabelWidth', 'configuredMaxLabelWidth', {
+    get() {
+      if (this.get('hasMaxLabelWidth')) {
+        return this.get('configuredMaxLabelWidth');
+      } else {
+        return null;
+      }
+    },
+    set(key, value) {
```

Comment:
It does, but actually this is the wrong fix. `maxLabelWidth` shouldn't be bound into the slider. It should be `configuredMaxLabelWidth`. That way nobody should be setting `maxLabelWidth` directly, so it won't need this set case